admin/client.go

```diff
@@ -169,6 +172,16 @@ func (c *Client) CreateProject(ctx context.Context, name string) (*types.Project
 		},
 	)
 	if err != nil {
+		st := status.Convert(err)
+		for _, detail := range st.Details() {
+			switch t := detail.(type) {
+			case *errdetails.BadRequest:
+				for _, violation := range t.GetFieldViolations() {
+					fmt.Printf("Invalid Fields: The %q field was wrong: %s\n", violation.GetField(), violation.GetDescription())
+				}
+			}
+		}
+
```

Comment:
So far, I/O has been handled by the CLI(packages in the `cmd`).

https://github.com/yorkie-team/yorkie/blob/f11a6ed1c207f55fed92f0cc0d16d027104afbf7/cmd/yorkie/document/list.go#L74-L84

For consistency, I think it's better to return an error here and handle the actual output from the CLI. 🤔

Reply:
Thanks. I changed to handle in the cmd.